src/main/java/Duke.java

```diff
@@ -1,10 +1,88 @@
+import java.lang.reflect.Array;
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+
+    public static void promptUser(String message) {
+        System.out.println(message);
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        //initiate objects
+        Scanner sc = new Scanner(System.in);
+        ArrayList<Task> list = new ArrayList<>();
+        //read user input
+        while(true) {
+            String input = sc.nextLine();
+            String[] words = input.split(" ");
+            String firstWord = words[0];
+            if (input.equals("bye")){
+                System.out.println("Bye. Hope to see you again soon!");
+                break;
+            } else if (input.equals("list")){
+                System.out.println("Here are the tasks in your list:");
+                for(int i = 0; i < list.size(); i++) {
+                    System.out.println(String.format("%d. %s", i+1, list.get(i)));
+                }
+            } else if (firstWord.equals("done")){
+                int num = Integer.parseInt(input.split(" ")[1]);
+                System.out.println("Nice! I've marked this task as done: ");
+                list.get(num-1).markAsDone();
+                System.out.println(list.get(num-1));
+            } else if (firstWord.equals("todo")) {
+                try {
+                    if(words.length == 1) {
+                        throw new StringIndexOutOfBoundsException("without argument");
+                    }
+                    Task todo = new ToDo(input.substring(5));
+                    list.add(todo);
+                    System.out.println("Got it. I've added this task:");
+                    System.out.println(todo);
+                    System.out.println(String.format("Now you have %d tasks in your list", list.size()));
```

Comment:
Maybe instead of repeating system.out.println(), you can combine them all with "+\n"? That could be cleaner!